user/storage.md

```diff
@@ -41,37 +41,22 @@ The `shared` directory is not intended as a way for hub users to share data with
 The recommended way to move code in and out of the hub is via git / GitHub.
 You should clone your project repo from the terminal and use git pull / git push to update and push changes.
 In order to push data to GitHub from the hub, you will need to set up GitHub authentication.
-This is a very quick guide to getting your GitHub authentication set up,
-adopted from the [Carpentries GitHub Remotes lesson](https://swcarpentry.github.io/git-novice/07-github/index.html#ssh-background-and-setup).
-
-1. Open a terminal in JupyterHub
-1. Type the command
-   ```
-   ssh-keygen -t ed25519 -C "YOUR EMAIL ADDRESS GOES HERE"`
-   ```
-   (Don't just copy this text; you have to put in tour actual email address in between the quotes.) This command will create an ssh public / private key pair.
-1. Enter a password for your new SSH key and record it in a safe place.
-   This password is used to "lock" the SSH key. It can't be used without the password.
-1. Type the command
-   ```
-   cat ~/.ssh/id_ed25519.pub
-   ```
-   and copy the result. It should look something like `ssh-ed25519 {long random string} {your email address}`.
-1. Go to <https://github.com/settings/keys>. Click the green  button that says "New SSH Key".
-   Give your key the title "JupyterHub SSH Key for Research Computing" and paste the
-   public key from the previous step into the "Key" box.
-1. Verify that your key works by typing
-   ```
-   ssh -T git@github.com
-   ```
-   on the command like of the Hub. (Note you will have to enter your SSH key password from step 3.)
-   This will return a message of the following form
-   ```
-   Hi {username}! You've successfully authenticated, but GitHub does not provide shell access.
-   ```
-   If you see that, it works! 🚀
-
-You should now be able to push to GitHub from the hub.
+[gh-scoped-creds](https://github.com/yuvipanda/gh-scoped-creds/) should be already setup
+on your 2i2c managed JupyterHub, and we shall use that to authenticate to GitHub for
+push / pull access.
+
+Open a terminal in JupyterHub, run `gh-scoped-creds` and follow the prompts.
```

Comment:
I just tried this on https://leap.2i2c.cloud/, but the command was not found. What will it take to get this actually deployed in our hub?

Reply:
@rabernat https://github.com/2i2c-org/docs/pull/138#discussion_r853909997 should show you how to test this one.

I'll deploy the settings to leap hub too shortly.

Reply:
Sorry for not reading the other comments which explained the situation very clearly. Trying now.

Reply:
https://github.com/2i2c-org/infrastructure/pull/1221 sets up the config for LEAP hub, but not deployed yet. Still needs us to merge https://github.com/pangeo-data/pangeo-docker-images/pull/305 to remove the manual `%pip` step.

Reply:
@rabernat did it work for you? Do you think we should add more docs here other than just 'follow the prompts'? Can also improve the prompts themselves of course

Reply:
I _just now_ managed to try it on staging. I pip installed and followed the instructions, and it worked! 🎉 

Then I hit a snag: I tried to clone a private repo using an SSH url (`git@github.com:rabernat/planets.git`). It tried it use my ssh key. Then I tried again with the HTTPS url (`https://github.com/rabernat/planets.git`) and it totally worked.

Should this approach only work with HTTPS git urls? If so, we need to say that clearly in the docs.

Reply:
The message

> Visit https://github.com/apps/pangeo-gcp-hub-push-access to manage list of repositories you can push to from this location

was also helpful. But I'd consider adding more details about this part to the 2i2c docs.

Reply:
@rabernat ah, good points! yes, it only works with https URLs, not ssh URLs. Fixed in https://github.com/yuvipanda/gh-scoped-creds/pull/24

Reply:
@rabernat good to know re: managing list of repos is more docs. I will add that to this PR.

Reply:
I've opened https://github.com/2i2c-org/infrastructure/pull/1223 to bring in the package installed into the image.

Reply:
@rabernat ok, this has been deployed onto leap and the pangeo GCP hub! I'll amend the docs in this PR today

Reply:
ok, I've added a little more docs there now.